lib/common/util.js

```diff
@@ -360,6 +362,14 @@ function makeAuthenticatedRequestFactory(config) {
     if (stream) {
       return stream;
     }
+
+    return {
+      abort: function() {
+        if (activeRequest_) {
+          activeRequest_.abort();
```

Comment:
Should we set `activeRequest_` to `null` after aborting to prevent multiple `abort()` calls?